app/src/main/java/com/example/capsule/ui/itemDetails/ItemDetailsFragment.kt

```diff
@@ -117,8 +140,27 @@ class ItemDetailsFragment : Fragment() {
         var category = categorySpinner.selectedItem.toString()
```

Comment:
I think you can change var to val since it is never changed for all lines 139 to 148